content/docs/04.5-devx/05-app-profile/15-db-services/4-connectivity.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Service Connectivity"
+metaTitle: "Palette Dev Engine Database Connectivity"
+metaDescription: "Palette Dev Engine Database Connectivity"
+hideToC: false
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+
+
+# Service Connectivity
+
+Connectivity is the process of establishing communication between the database service and clients. Connectivity is required to send SQL queries from clients to the database. Palette database services expose output variables that contain the required connectivity information such as the hostname, database user, user password, and database port. You can use these output variables to connect your application to the database.
```

Comment:
@resmaa - this has still evidences of database keyword into this page. Additionally, I have some thoughts on organizing this doc. Lets meet for few minutes and discuss on the ordering.